src/nexus.h

```diff
@@ -41,18 +41,27 @@ class Nexus : public QObject
 {
     Q_OBJECT
 public:
-    void start();
-    void showMainGUI();
-
     static Nexus& getInstance();
     static void destroyInstance();
-    static Core* getCore();
-    static Profile* getProfile();
-    static void setProfile(Profile* profile);
-    static Widget* getDesktopGUI();
     static QString getSupportedImageFilter();
     static bool tryRemoveFile(const QString& filepath);
 
+    void start();
+    void showMainGUI();
+
+public:
+    inline Widget* getDesktopGUI() const
+    {
+        return widget;
+    }
+
+    inline Profile* getProfile() const
```

Comment:
I just don't like methods in `.h` file. If you think, that it's make code better -- ok.